srcs/test/integration/misc_test.py

```diff
@@ -1,7 +1,39 @@
+import psutil
 import requests
 
 # MISC
 def test_method_not_implemented_501():
     url = 'http://localhost:8080/'
     response = requests.options(url)
     assert response.status_code == 501
+
+def test_invalid_cgi_binary_500():
+    url = 'http://localhost:8094/info.php'
+    response = requests.get(url)
+    assert response.status_code == 500
+
```

Comment:
Great you added some tests but I'd made a custom `cgi_test` file (I understand you've placed them here in order to gather all error test cases)